examples/http_client_example.py

```diff
@@ -0,0 +1,102 @@
+"""
+Copyright © SurrealDB Ltd
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+import asyncio
+# import the http client this client is not as feature rich as the websocket client
```

Comment:
separate these 2 sentences onto 2 lines i think. the second sentence likely isn't even necessary and can be removed